cliquet/initialization.py

```diff
@@ -48,6 +48,9 @@ def setup_version_redirection(config):
     redirect_enabled = settings['cliquet.version_prefix_redirect_enabled']
     version_prefix_redirection_enabled = asbool(redirect_enabled)
 
+    route_prefix = config.route_prefix
+    config.add_settings(route_prefix=route_prefix)
```

Comment:
It sounds to me that if `route_prefix` is set in the `settings` then it should not be set directly on the `config` object as done currently. This was a hack anyway.

And if you set `route_prefix` in the `settings` you may want to use the name `cliquet.route_prefix` to avoid clashes with application-level names.